src/structs.jl

```diff
@@ -42,10 +42,28 @@ struct EmptyCell <: AbstractCell
     ref::CellRef
 end
 
+abstract type AbstractCellDataFormat end
+struct EmptyCellDataFormat <: AbstractCellDataFormat end
+"""
+Keeps track of formatting information.
+"""
+struct CellDataFormat <: AbstractCellDataFormat
+    id::Integer
+
+    function CellDataFormat(id::Integer)
+        @assert id >= 0
+        new(id)
+    end
+end
+
+const CellValueTypes = Union{String, Missings.Missing, Float64, Int, Bool, Dates.Date, Dates.Time, Dates.DateTime}
 """
 CellValue is a Julia type of a value read from a Spreadsheet.
 """
-const CellValue = Union{String, Missings.Missing, Float64, Int, Bool, Dates.Date, Dates.Time, Dates.DateTime}
+struct CellValue
+    value::CellValueTypes
+    styleid::AbstractCellDataFormat
+end
```

Comment:
Should this have another constructor like this:
```julia
CellValue(value::CellValueTypes) = ...
```
which just uses the default style ID?